src/app/clients/ActionBar/Actions.tsx

```diff
@@ -0,0 +1,261 @@
+"use client";
+
+import React, { useState } from "react";
+import Menu from "@mui/material/Menu/Menu";
+import MenuList from "@mui/material/MenuList/MenuList";
+import MenuItem from "@mui/material/MenuItem/MenuItem";
+import dayjs, { Dayjs } from "dayjs";
+import { ClientsTableRow } from "@/app/clients/getClientsTableData";
+import ActionsModal, {
+    DayOverviewInput,
+    DriverOverviewInput,
+} from "@/app/clients/ActionBar/ActionsModal";
+import {
+    DayOverviewModalButton,
+    DriverOverviewModalButton,
+    ShippingLabelsModalButton,
+    ShoppingListModalButton,
+} from "@/app/clients/ActionBar/ActionsModalButton";
+import { SelectChangeEvent } from "@mui/material";
+
+const isNotMoreThanOne = (value: number): boolean => {
+    return value < 1;
```

Comment:
This expression doesn't match the name: `value < 1` would be "isNotAtLeastOne".

(That said, the 'not' boolean naming conventions on these consts aren't easy to read in any case.)

Reply:
I'm not going to fix that in this PR though. I've raised VFB-53 (Action behaviours don't match functional spec), to cover doing a full investigation of Actions.